package.json

```diff
@@ -182,6 +182,11 @@
 						"light": "#19a1d6",
 						"highContrast": "#19a1d6"
 					}
+				},
+				"cursorless.individualHatAdjustments": {
```

Comment:
a bit on the fence about having this setting.  I added it because it was really useful while prototyping rather than restarting the extension every time